index.org

```diff
@@ -843,14 +843,14 @@ helper-functions:
   (let [obj (js->clj o :keywordize-keys true)]
     (if (map? obj)
       (change-keys obj ->kebab-case)
-      (if (seq? obj)
+      (if (vector? obj)
         (map flowjs->clj obj)
         obj))))
 
 (defn- clj->flowjs
   "Convert Clojure map into a JavaScript object with snake_cased keys."
   [o]
-  (->> (change-keys o ->snake_case)
+  (->> (change-keys o ->camelCase)
        (clj->js)))
```

Comment:
Now it is "camelCased keys"?

Reply:
Yes, it was wrong at first. React-flow (the base library) expects the data to be in camel case

Reply:
Yeah, we should also change in the comment "snake_cased keys" -> "camelCased keys"